backends/graphics/surfacesdl/surfacesdl-graphics.h

```diff
@@ -284,6 +284,7 @@ class SurfaceSdlGraphicsManager : public SdlGraphicsManager {
 			// desiredAspectRatio set to (0, 0) by AspectRatio constructor
 			filtering = false;
 
+			mode = GFX_SURFACESDL;
```

Comment:
How is this struct member used?

Reply:
It allows subclasses of SurfaceSdlGraphicsManager to define extra graphics modes in addition to the standard one.